src/main/pages/user-management/organizations.adoc

```diff
@@ -11,49 +11,66 @@ folder: user-management
+A list of all the organizations displays on the *Organization* page. The list contains the following general information for each organization: number of members, total RAM, available RAM, and number of sub-organizations.
 
-[id="organization-list"]
-== Organization List
+[id="adding-members-to-organizations"]
+== Adding members to organizations
 
-The list of all organizations are displayed on organizations page. The list contains the general information for each organization: number of members, total and available RAM and number of sub-organizations.
+To add members to an organization:
 
-[id="adding-organization-members"]
-== Adding Organization Members
+. Click the *Add* button to add a member.  A new pop-up window displays. 
 
-Adding organization members by clicking the "Add" button will display a new popup. You can change an organization member’s role or remove them from the organization at any time.
+. Enter the new member name.  
 
-Note: Users with the green checkmark beside their name already have an account on your Eclipse Chesystem and will be added to the organization. Users without a checkmark do not have an account and will not be added into the organization.
+[NOTE]
+====
+Users with the green checkmark beside their name already have an Eclipse Che account and can be added to the organization. Users without a checkmark do not have an account and cannot be added into the organization.
+====
 
-[id="workspaces-in-organization"]
-== Workspaces in Organization
+[id="workspaces-in-organizations"]
+== Workspaces in organizations
 
-Workspace is created inside of an organization and uses the resources of this very organization. Workspace creator has to choose the organization on workspace creation page.
+A workspace is created inside of an organization and uses the resources of the organization. The workspace creator chooses the organization on the *Workspace Creation* page.  
 
-[id="email-notifications"]
-== Email Notifications
+[id="setting-email-notifications"]
+== Setting email notifications
 
-When a user joins or leaves an organization, Che server can send a notification in case SMTP is properly configured in `che.env` (or by adding envs to deployment in case fo OpenShift). By default, Che does not offer any built-in SMTP server. You may use any mail server you want. Below is an example on how to use own Gmail to send notification emails:
+When a user joins or leaves an organization, Che server sends a notification. You either configure SMTP in `che.env` or for OpenShift you add environment variables to the deployment to send a notification.  Che does not have any built-in SMTP server by default. You may use any mail server.
```

Comment:
"You either configure SMTP...."

This sentence is jumbled and confusing. I'd suggest rewriting it. More than one sentence may clarify the intent.